src/buildList.ts

```diff
@@ -0,0 +1,35 @@
+import { processTokens } from './processTokens';
+import packageJson from '../package.json';
+import { WrapperMap } from './types';
+
+export async function buildList(
+    skipImageProcessing = false,
+): Promise<Partial<WrapperMap>> {
```

Comment:
Shouldn't need Partial here? The WrapperMap type should indicate which properties are optional correctly (and be changed if I didn't do so)